### DISCUSSION TITLE
New Relic Buildpack release

### DISCUSSION THREAD

**Discussion by michelklein** [paketo-buildpacks/feedback#225] [General]
Hello everyone,
We are eagerly awaiting the nex New Relic Buildpack release. We have migrated our Spring Boot projects to Java 21 and therefore need the latest New Relic Java Agent. It has now been a month since the last release. When do you plan to release the latest version?

Thanks, Michèl

**Comment by dmikusa** [comment]
We don't have a fixed release cycle for this buildpack. I would say that we typically release about once a month, but it could be more frequent, hopefully not less frequent.

Anyway, always feel free to ask for a release. Happy to cut one if there's a need. Just cut https://github.com/paketo-buildpacks/new-relic/releases/tag/v8.4.0 for you.
